thoth/storages/result_base.py

```diff
@@ -84,12 +86,38 @@ def connect(self) -> None:
         """Connect the given storage adapter."""
         self.ceph.connect()
 
-    def get_document_listing(self) -> typing.Generator[str, None, None]:
-        """Get listing of documents available in Ceph as a generator."""
-        for document_id in self.ceph.get_document_listing():
-            # Filter out stored requests.
-            if not document_id.endswith(".request"):
-                yield document_id
+    def _iter_dates_prefix_addition(
+        self, start_date: date, end_date: typing.Optional[date] = None
+    ) -> typing.Generator[str, None, None]:
+        """Create prefix based on dates supplied."""
+        if end_date is None:
+            end_date = date.today() + timedelta(days=1)  # Today inclusively.
+
+        walker = start_date
+        step = timedelta(days=1)
+        while walker < end_date:
+            yield walker.strftime(f"{self.RESULT_TYPE}-%y%m%d")
+            walker += step
+
+    def get_document_listing(
```

Comment:
missing `get_document_count` to have the new parameters

Reply:
Add, PTAL 👍🏻